examples/js/loaders/OBJLoader.js

```diff
@@ -40,315 +65,431 @@ THREE.OBJLoader.prototype = {
 
 	},
 
-	parse: function ( text ) {
-
-		console.time( 'OBJLoader' );
+	_createParserState : function()
```

Comment:
Sure, its not of any utility outside the parse function. I guess a wrapping function so the function is only created once would be in place.

The state object itself can be useful if you want to do some custom stuff. First comes to mind if you want to start a new object from a special comment line. That can be arraged by passing the state into the callback (like discussed for comments).

```js
if (isCommenLine && typeof options.onCommentLine === "function") {
    options.onCommentLine(state, line);
    continue
}
```